openshift/configmap-template.yaml

```diff
@@ -31,6 +31,7 @@ objects:
     middletier-pod-cpu-limit: '0.5'
     middletier-pod-memory-limit: '0.5Gi'
     backend-namespace: ${THOTH_BACKEND_NAMESPACE}
+    infra-namespace: ${THOTH_INFRA_NAMESPACE}
```

Comment:
why do we need it?

Reply:
User API asks for template for adviser/solver/analyzer from there and runs it (instead of hardcoding templates into sources).